src/vendor-dashboard/reports/analytics/components/leaderboard/data/top-selling-products-mock-data.js

```diff
@@ -0,0 +1,94 @@
+/*
+The data returned by /reports/products will contain a product_id ref
+And as such will require data layer logic for products to fully build the table
+
+[
+  {
+    "product_id": 20,
+    "items_sold": 100,
+    "net_revenue": 999.99,
+		"orders_count": 54,
+		"name": 'Product name',
+    "_links": {
+      "product": [
+        {
+          "href": "https://example.com/wp-json/wc-analytics/products/20"
+        }
+      ]
+    }
+  }
+]
+
+*/
+export default [
+	{
+		product_id: 20,
+		items_sold: 1000,
+		net_revenue: 999.99,
+		orders_count: 54,
+		name: 'awesome shirt',
+		_links: {
+			product: [
+				{
+					href: 'https://example.com/wp-json/wc-analytics/products/20',
+				},
+			],
+		},
+	},
+	{
+		product_id: 22,
```

Comment:
Why are we using static products as default? Pls leave it empty array as default